apps/docs/src/components/landing/TitleImage.tsx

```diff
@@ -0,0 +1,82 @@
+import { Button } from '@uzh-bf/design-system'
+
+export function TitleImage() {
+  return (
+    <div className="bg-white">
+      <div className="relative">
+        <div className="mx-auto max-w-7xl">
+          <div className="relative z-10 pt-14 lg:w-full lg:max-w-2xl">
+            <svg
+              className="absolute inset-y-0 right-8 hidden h-full w-80 translate-x-1/2 transform fill-white lg:block"
+              viewBox="0 0 100 100"
+              preserveAspectRatio="none"
+              aria-hidden="true"
+            >
+              <polygon points="0,0 90,0 50,100 0,100" />
+            </svg>
+
+            <div className="relative px-6 py-12 sm:py-40 md:py-32 lg:px-8 lg:py-56 lg:pr-0">
+              <div className="mx-auto max-w-2xl lg:mx-0 lg:max-w-xl">
+                <div className="hidden sm:mb-10 sm:flex">
+                  <div className="text-md relative rounded-full px-3 py-1 leading-6 text-gray-500 ring-1 ring-gray-900/10 hover:ring-gray-900/20">
+                    KlickerUZH v3.1 has been released with brand new features{' '}
```

Comment:
As we're working on this anyway - we could already switch this to KlickerUZH v3.2 (has been released yesterday) - we should have a community post to be linked in the coming days